Package.swift

```diff
@@ -9,7 +9,10 @@ let package = Package(
     products: [
         .library(
             name: "KlaviyoSwift",
-            targets: ["KlaviyoSwift"])
+            targets: ["KlaviyoSwift"]),
+        .library(
+            name: "KlaviyoSwiftExtension",
```

Comment:
Not really married to this name but I think this follows the existing convention.

Reply:
Would there be a situation where we need multiple extensions? Maybe we want to be more clear what this extension does if its just going to be for handling rich push. Otherwise, I think the name seems fine

Reply:
Yeah @evan-masseau and I were having the same conversation earlier today and decided to keep the name as is since even in a future where we support multiple extensions we may not want to create a new target for each but instead use this as a general purpose target to support all extensions related support outside of the core SDK.